openwisp_firmware_upgrader/static/firmware_build/css/upgrade_build.css

```diff
@@ -0,0 +1,11 @@
+input.upgrade-build {
```

Comment:
I would call this `input.object-tools-button` so it could be reused in this application if needed, without having to change its name